src/Core/src/Handlers/Label/LabelHandler.Android.cs

```diff
@@ -30,5 +32,28 @@ public static void MapTextColor(LabelHandler handler, ILabel label)
 		{
 			handler.TypedNativeView?.UpdateTextColor(label, DefaultTextColor);
 		}
+
+		public static void MapFontFamily(LabelHandler handler, ILabel label)
+		{
+			MapFont(handler, label);
+		}
+
+		public static void MapFontSize(LabelHandler handler, ILabel label)
+		{
+			MapFont(handler, label);
+		}
+
+		public static void MapFontAttributes(LabelHandler handler, ILabel label)
+		{
+			MapFont(handler, label);
+		}
+
+		static void MapFont(LabelHandler handler, ILabel label)
+		{
+			var context = MauiApp.Current?.Context ?? throw new InvalidOperationException($"The MauiApp.Current.Context can't be null.");
+			var fontManager = context.Services.GetRequiredService<IFontManager>();
```

Comment:
This is sus, but for now it is the only way to get the `IMauiContext`. I am doing it in the handler to reduce the junk in the extension methods and other places.